packages/web-react/src/hooks/__tests__/useStyleUtilities.test.ts

```diff
@@ -150,4 +150,22 @@ describe('useStyleUtilities hook', () => {
       'text-desktop-right',
     ]);
   });
+
+  it('should not process null or undefined style utilities', () => {
```

Comment:
Edge case, is it possible to pass an empty string? If so, shouldn't we rather check it as well?

Reply:
You can shorten `null or undefined` to just `nullish`.

Reply:
@dlouhak Good point and nice catch. The empty string also is problematic, so I have refactored the check a bit more.